### PR TITLE
Bed Rush - Play Testing Updates (Observer Kits/Alerts, etc.)

### PR DESCRIPTION
**Pull request by arcadeboss**
Didn't realize that phase being used to block play of maps, updated so map can be tested.

### CODE REVIEW

File: arcade/standard/bed_rush/map.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <map proto="1.4.2">
 <name>Bed Rush</name>
-<version>1.0.0</version>
-<phase>development</phase>
+<version>1.0.1</version>
+<phase>production</phase>
```
Comment (TTtie) [inline arcade/standard/bed_rush/map.xml:4]: Maybe staging phase would be better suited for the time being?
```suggestion
<phase>staging</phase>
```

**Comment by Pablete1234** [comment]
This map is still very much a map in development, get a mapdev to playtest it before you move it to a different stage

**Comment by arcadeboss** [comment]
> This map is still very much a map in development, get a mapdev to playtest it before you move it to a different stage

No map devs tend to be around in the morning when the player count matches the size of this map.

**Comment by arcadeboss** [comment]
Mame has suggested to change it to staging (like TTtie, thanks) so I've done that.

**Comment by arcadeboss** [comment]
These were the issues logged by Pablo during game testing on 8/9, and our fixes:

_- Obs spawn is off by half a block_

Improved the location of observer, mainly adding pitch and adjusting centering

_- Obs are getting affected by kits (inv cleared, and kit given)_

Included participating in the filter for kits to avoid observers

_- Obs do not get message/title of round ending_

Restructured round end alerts to go to everyone (obs, team red, team blue)
Observers hear the winning sounds

_- The 3,2,1 message is off-by-1 in timing from the bossbar of next round starts in X_

Added a .3 second delay in the boss bar to better match the main game timing

_- After match end, it still triggers a 3,2,1 title thing as if there was a next round_

Never saw this, but added match not finished filter to keep game state bounded.

**Comment by arcadeboss** [comment]
Updated from further comments from Pablo.  The countdown sequence is now more synchronized, and switched not-match-finished with match-running.

**Comment by arcadeboss** [comment]
More updates from suggestions coming, but needs to be play tested first.

**Comment by arcadeboss** [comment]
Completed play testing on XML simplification, and pushed changes in last commit.

**Pull request merged**